templates/footerSection.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="footer__item">
+   <div class="footer__icon">
+     <img class="footer__icon--image" src="{{assetSrc icon}}" role="presentation" alt="Icon of {{title}}">
```

Comment:
If the role is presentation, the alt text should be empty since the image will be decorative, otherwise screen readers will reach the image and will announce it